Bio/SeqIO/UniprotIO.py

```diff
@@ -455,6 +455,12 @@ def _parse_feature(element):
             feature.type = element.attrib.get("type", "")
             if "id" in element.attrib:
                 feature.id = element.attrib["id"]
+            if "description" in element.attrib:
+                feature.qualifiers["description"] = element.attrib["description"]
+            if "evidence" in element.attrib:
+                feature.qualifiers["evidence"] = element.attrib["evidence"]
+            if "status" in element.attrib:
+                feature.qualifiers["status"] = element.attrib["status"]
```

Comment:
I'm looking at that and thinking a for loop would be more elegant - especially if there might be more extra fields later on?

```python
for field in ["description", "evidence", "status"]:
    if field in element.attrib:
        feature.qualifiers[field] = element.attrib[field]
```

Or, if those are almost always present perhaps this:

```python
for field in ["description", "evidence", "status"]:
    try:
        feature.qualifiers[field] = element.attrib[field]
    except KeyError:
        pass
```

That is longer though, so maybe not?

Reply:
I wonder if we can make it even more general so new attributes don't require a code change at all. 

```python
for field in [k for k in element.attrib.keys() if k not in ["id", "type"]]:
    feature.qualifiers[field] = element.attrib[field]
```

Reply:
even the if statment for the id can be removed:

```python
            for field in [k for k in element.attrib.keys() if k != "type"]:
                feature.qualifiers[field] = element.attrib[field]
```

I wonder if type is that special that it has to be set even if it is not in the attribute list?

Reply:
OK, it was all allready there... just a few lines up

https://github.com/biopython/biopython/blob/46a5c82e126713d22894301ba356c71e4c03732d/Bio/SeqIO/UniprotIO.py#L453-L454